### ISSUE TITLE
Ordering of output in sentiment_pipe function

### ISSUE THREAD

**Issue by kaushalshetty** [lvwerra/trl#117]
In the notebook gpt2-sentiment-control.ipynb (Optimize model section) , 
`logits = [torch.tensor(output[1]["score"]) for output in sentiment_pipe(texts, **sentiment_pipe_kwargs)]`

Why do we store output[1]["score"] as the reward? I assumed that "We will use the logits for positive class as a reward signal for the language model." but does sentiment_pipe always has the output[1] as an index for the the positive class?

**Comment by lvwerra** [comment]
I just checked and you are right, this is not always the case! I will fix the example. cc @younesbelkada maybe this explains the spikes.
